.storybook/storybook.css

```diff
@@ -0,0 +1,4 @@
+.sb-show-main {
+  /* overrides `overflow: hidden;` on body */
+  overflow: visible;
+}
```

Comment:
OMG THANK YOU

Reply:
do we need to keep the commented out line?

Reply:
I wouldn't say that we _need_ to keep it, but I added it to give some context since it's the only thing in the whole file and the class name doesn't really give us much info about what it's for either

Reply:
I did not have enough coffee in my system when reading. I agree this is good context to have.